modules/web/src/app/core/services/auth/guard.ts

```diff
@@ -103,3 +104,23 @@ export class AuthzGuard implements CanActivate {
     return from(this._router.navigate([View.Projects]));
   }
 }
+
+@Injectable()
+export class isEnabledPathGuard implements CanActivate {
```

Comment:
This is not related to authentication so please move this to a separate file, perhaps create a directory named `ssh-key` and move this in `guard.ts` file there. You can also move `ssh-key.ts` inside that directory to avoid confusion.
As for the naming, I would just call it `SSHKeyGuard`.